doc/en/user/source/services/wps/administration.rst

```diff
@@ -34,6 +34,7 @@ Resource settings:
 
   * *Resource expiration timeout*: number of seconds the result of a asynchronous execution will be kept available on disk for user to retrieve. Once this time is expired these resources will be eligible for clearing (which happens at regular intervals).
   * *Resource storage directory*: where on disk the input, temporary and output resources associated to a certain process will be kept. By default it will be the ``temp/wps`` directory inside the GeoServer data directory
+  * *External output directory*: where on disk, certain processes are allowed to write their outputs outside of the resource storage so that the files will not be automatically removed based on the resource expiration timeout. Leave empty to disable writing outside of the resource storage.
```

Comment:
Is this option only available for process that write files? Or is it for any async response ...

Reply:
Oh I see: difference between "resource storage" and "external output" directory.

Still does not quite answer my question about async output. Consider using: "response" and "result" rather than "output".

```
  * :guilabel:`Resource expiration timeout`: number of seconds the result of an asynchronous execution will be kept available on disk for user to retrieve. Once this time is expired these resources will be eligible for clearing (which happens at regular intervals).
  * :guilabel:`Resource storage directory`: where on disk the temporary input resources, temporary outputs resources, and asynchronous results of a process execution will be kept. By default it will be the :file:`temp/wps` directory inside the GeoServer data directory. These resources are subject to the :guilabel:`Resource expiration timeout` above.
  * :guilabel:`External output directory`: Some processes allow execution outputs to be stored in an external output directory (not subject :guilabel:`Resource expiration timeout`). To enable this functionality provide a path to external storage with the understanding that you are responsible for managing the contents of this folder. Leave empty to disable writing outside of the resource storage.
```

Reply:
The documentation was updated based on your feedback.

Reply:
Thanks!